<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-14L0LB4" failures="1" tests="4" name="Test" time="73.839" errors="0" timestamp="2022-08-11T10:46:10 IST">
  <testcase classname="oneClick.oneClickTDD.oneClickLogin" name="loginOneClick" time="13.868"/>
  <testcase classname="oneClick.oneClickTDD.oneClickLogin" name="verifyLoginPageTitle" time="0.526">
    <failure type="java.lang.AssertionError" message="expected [Log in to get start] but found [Log in to get started]">
      <![CDATA[java.lang.AssertionError: expected [Log in to get start] but found [Log in to get started]
at oneClick.oneClickTDD.loginPage.verifyLoginPageTitle(loginPage.java:65)
at oneClick.oneClickTDD.oneClickLogin.verifyLoginPageTitle(oneClickLogin.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginPageTitle -->
  <testcase classname="oneClick.oneClickTDD.oneClickLogin" name="resetLoginPage" time="9.955"/>
  <testcase classname="oneClick.oneClickTDD.oneClickLogin" name="forgetPassword" time="4.937"/>
</testsuite> <!-- Test -->
